src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -420,6 +444,34 @@ private static void close(final Closeable res) {
             this.output = out;
             this.level = lvl;
         }
+
+        /**
+         * Checks if line is part of a stack trace and should be appended.
+         * @param string String to check
+         * @return boolean result
+         */
+        private static boolean shouldAppend(final String string) {
+            final String leftStrip = stripStart(string);
+            return leftStrip.startsWith(PREFIX_AT) || leftStrip.startsWith(PREFIX_CB) || leftStrip.startsWith(PREFIX_DOTS);
```

Comment:
@dean-e-clark line cannot have more than 80 chars, please read http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#indentation